build.sbt

```diff
@@ -1459,6 +1477,7 @@ lazy val `sbt-scripted-library` = (project in file("dev") / "sbt-scripted-librar
 
 lazy val `service-locator` = (project in file("dev") / "service-registry" / "service-locator")
   .settings(runtimeLibCommon: _*)
+  .settings(noMima)
```

Comment:
Published artifact.

Reply:
Of all the artifacts @marcospereira mentioned this is the one I have no doubts about. `service-locator` should not use `noMima`.

Reply:
Why?  It's a dev-mode dependency, not a production code dependency.